webpack/scenes/Subscriptions/Manifest/DeleteManifestModalText.js

```diff
@@ -0,0 +1,11 @@
+const DeleteManifestModalText = `<p>Are you sure you want to delete the manifest?</p>
+<p>Note: Deleting a subscription manifest is STRONGLY discouraged. Deleting a manifest will:</p>
+<ul class="list-aligned">
+  <li>Delete all subscriptions that are attached to running hosts.</li>
+  <li>Delete all subscriptions attached to activation keys.</li>
+  <li>Disable Red Hat Insights</li>
+  <li>Require you to upload the subscription-manifest and re-attach subscriptions to hosts and activation keys.</li>
+</ul>
+<p>This action should only be taken in extreme circumstances or for debugging purposes.</p>`;
```

Comment:
How about splitting it into three parts: "Are you sure..", "Note: ..." and "This action..." to get at least the paragraphs out of the translated text.